src/core/prompt.rs

```diff
@@ -1,8 +1,30 @@
 use io::file::FileStatus;
-use state::editor::Editor;
+use io::redraw::RedrawTask;
+use state::editor::{BufferInfo, BufferManager, Editor};
+use edit::buffer::{Buffer, SplitBuffer};
 
 use std::process::exit;
 
+enum BufferCommand {
+    SwitchToBuffer(usize),
+}
+
+fn try_get_buffer_command(c: &str) -> Option<BufferCommand> {
```

Comment:
Yeah, I wrote it expecting that I would need special logic to handle the buffer commands that take a buffer id with them (like `b{n}` does). That's the only one that needs it now. Do you think I should just move this logic down into the call site?